components/media/lib.rs

```diff
@@ -20,12 +20,14 @@ pub use crate::media_channel::glplayer_channel;
 use crate::media_channel::{GLPlayerChan, GLPlayerPipeline, GLPlayerReceiver, GLPlayerSender};
 use crate::media_thread::GLPlayerThread;
 
+type GLPlayerSenderType = GLPlayerSender<(u32, Size2D<i32>, usize)>;
+
```

Comment:
I thin in this case the original is a bit better as it's hard to understand what the sender sends now.